src/components/HomeHeader.js

```diff
@@ -68,6 +68,11 @@ class HomeHeader extends React.Component {
           <a href = "https://docs.litmuschaos.io/" target = "_blank" rel="noopener noreferrer">
           <h3 className="top-header-contribute">Docs<span className="contribute-icon-container"></span></h3>
           </a>
+          <a href = "https://app.slack.com/client/T09NY5SBT/CNXNB0ZTN" target = "_blank" rel="noopener noreferrer">
```

Comment:
this URL will not work
@aditya109  can you contact to the marketing team, and get the correct URL